libraries-5/src/test/java/com/baeldung/githubapi/RepositoryUnitTest.java

```diff
@@ -0,0 +1,95 @@
+package com.baeldung.githubapi;
+
+import com.google.common.base.Charsets;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.Test;
+import org.kohsuke.github.*;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class RepositoryUnitTest {
+    @Test
+    void whenWeListAUsersRepositories_thenWeCanAccessTheRepositories() throws IOException {
+        GitHub gitHub = GitHub.connectAnonymously();
+
+        GHUser user = gitHub.getUser("eugenp");
+        List<GHRepository> repositoriesList = user.listRepositories().toList();
+        assertEquals(6, repositoriesList.size());
```

Comment:
These kind of assertions will not be stable (can't rely on it always being 6)

Reply:
Is it better to just change this to a log then? Or is it ok to have unstable assertions if I change it to a LiveTest?